app/components/Header.tsx

```diff
@@ -132,6 +133,15 @@ export function Header() {
                       >
                         Peer Endorsements
                       </Link>,
+                      idp === 'COGNITO' ? (
```

Comment:
React generally supports `null`, `false`, and `undefined` as no-ops, so it is safe to use short-circuit evaluation here. Replace this with

```jsx
idp === 'COGNITO' && (
  <Link>
     ...
  </Link>
),
```